electrumx/server/mempool.py

```diff
@@ -328,7 +328,11 @@ def deserialize_txs():    # This function is pure
                 # mempool or it may have gotten in a block
                 if not raw_tx:
                     continue
-                tx, tx_size = deserializer(raw_tx).read_tx_and_vsize()
+                try:
+                    tx, tx_size = deserializer(raw_tx).read_tx_and_vsize()
+                except LitecoinMWTxError:
```

Comment:
It's probably cleaner to define a `SkipException` here and use that instead, to not leak this coin-specific error definition in the generic mempool processor. Then the exception can also contain more verbose information that can be logged.

Reply:
> to not leak this coin-specific error definition in the generic mempool processor

indeed

> define a `SkipException`

or maybe `SkipTxDeserialize`